UI/window-basic-settings.cpp

```diff
@@ -291,6 +292,9 @@ OBSBasicSettings::OBSBasicSettings(QWidget *parent)
 	ui->setupUi(this);
 
 	setWindowFlags(windowFlags() & ~Qt::WindowContextHelpButtonHint);
+	QScreen *screen = App()->GetMainWindow()->windowHandle()->screen();
```

Comment:
I'm not an expert in QT, but it looks like we could do the same with:

```cpp
        QScreen *screen = QGuiApplication::primaryScreen();
        const int screenHeight = screen->geometry().height();
        setMaximumHeight(screenHeight * 0.9);
```

Also, seems like the extra include `QWindow` is not needed.

Reply:
The reason for the somewhat convoluted fetching of the main app window's screen is that it reliably gets the "correct" screen, i.e. the screen the application is on.  Trying the same with the current window or with `QGuiApplication::primaryScreen` seems to give the wrong screen (or at least, _can_ give the wrong screen).  I am similarly inexpert in QT, and maybe there's some way to improve it, but for now this is reasonable.

Surprisingly, the QWindow include is needed in order to compile; otherwise you'll get an undefined type error.